compiler/nimconf.nim

```diff
@@ -238,6 +238,23 @@ proc getSystemConfigPath*(conf: ConfigRef; filename: RelativeFile): AbsoluteFile
     if not fileExists(result): result = p / RelativeDir"etc/nim" / filename
     if not fileExists(result): result = AbsoluteDir"/etc/nim" / filename
 
+const nimRepoDirCT = currentSourcePath / ".." / ".." # this should be exposed somewhere
+
+proc getNimRoot*(conf: ConfigRef): AbsoluteDir =
```

Comment:
We have this, it's called ``getSystemConfigPath``

Reply:
fixed; `getSystemConfigPath` did not work (eg `getSystemConfigPath(conf, "doc/nimdoc.css")` would return `/etc/nim/doc/nimdoc.css` on my system, which not only doesnt' exist but is not what I want here.
However, `getPrefixDir` works, both for installed nim as well as for a local development nim binary; so I've removed `getNimRoot`